### PR TITLE
7-post-api-reviews-review_id-comments

### PR DESCRIPTION
**Pull request by nasben64**

### CODE REVIEW

**Review by Harrison365 — CHANGES_REQUESTED** [review]
Hi Nasser :)
Please...
-You may want to switch author in req.body to username (like on the ticket)
-Test the entire return comment (e.g. that it has votes and correct review_id etc.)
-Test for present but incorrect username.

Thanks 👍

**Comment by nasben64** [comment]
hi Harrison 
I have fixed the suggested issues can you check again please

File: models/comments.js [hunk 1]
```diff
@@ -14,3 +14,17 @@ exports.selectCommentsByReviewId = (review_id) => {
...
+      .then(() => {
+        const queryStr = `INSERT INTO comments (body, author, review_id) 
+            VALUES ($1, $2, $3 ) RETURNING *;`;
+        return db.query(queryStr, [body, username, review_id]);
+      })
+      .then((result) => {
+        return result.rows[0];
+      });
+  });
+};
```
Comment (emilyb93) [inline models/comments.js]: a little bit of nesting here doing .then on checkUserExists here

**Review by emilyb93 — CHANGES_REQUESTED** [review]
un nest these promises, and this is good to go

**Comment by nasben64** [comment]
the nesting issue fixed by using Promise.all()

**Pull request merged**